### PR TITLE
Task 0003

### PR DESCRIPTION
**Pull request by marrodgar62**

### CODE REVIEW

File: src/main/java/com/ispp/heartforchange/controller/VolunteerController.java [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+			AuthenticationManager authenticationManager) {
+		super();
+		this.volunteerServiceImpl = volunteerServiceImpl;
+		this.jwtUtils = jwtUtils;
+		this.authenticationManager = authenticationManager;
+	}
+	
+	/*
+	 * Get all volunteer
+	 *
```
Comment (alvuberui) [inline src/main/java/com/ispp/heartforchange/controller/VolunteerController.java:55]: Parameter is missing in comment

File: src/main/java/com/ispp/heartforchange/controller/VolunteerController.java [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+	 */
+	@GetMapping
+	public ResponseEntity<?> getAllVolunteer(){
+		List<VolunteerDTO> volunteers = volunteerServiceImpl.getAllVolunteers();
+		return ResponseEntity.ok(volunteers);
+	}
+	
+	/*
+	 * Get all volunteer by ONG
+	 *
```
Comment (alvuberui) [inline src/main/java/com/ispp/heartforchange/controller/VolunteerController.java:66]: parameter comment is missing

File: src/main/java/com/ispp/heartforchange/service/impl/VolunteerServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+		for(Volunteer volunteer: volunteers) {
+			VolunteerDTO volunteerDTO = new VolunteerDTO(volunteer, volunteer.getHourOfAvailability(), volunteer.getSexCrimes());
+			volunteersDTO.add(volunteerDTO);
+		}
+		return volunteersDTO;
+	}
+	
+	/*
+	 * Get volunteer by ong
+	 * @Params Long id
```
Comment (alvuberui) [inline src/main/java/com/ispp/heartforchange/service/impl/VolunteerServiceImpl.java:57]: comment is wrong

**Review by alvuberui — APPROVED** [review]

**Review by fraorober — APPROVED** [review]
Very good, there are not problems.

**Pull request merged**